support/@trustfractal/polkadot-utils/src/lib/watcher.ts

```diff
@@ -12,8 +12,8 @@ export class TxnWatcher {
   public status: AnyJson | string = 'Unsubmitted';
 
   private onReady = new MultiCallback<void>();
-  private onInBlock = new OnceMultiCallback<void>('onInBlock');
-  private onFinalized = new OnceMultiCallback<void>('onFinalized');
+  private onInBlock = new OnceMultiCallback<string>('onInBlock');
```

Comment:
These should be an object so introducing additional data isn't a breaking change. Something like:

```ts
export interface TxnInBlock {
  block: string,
  txnHash: string,
}
```

But we only need the block hash right now. Same for OnFinalized.